packages/beacon-node/src/chain/blocks/verifyBlocksExecutionPayloads.ts

```diff
@@ -236,6 +239,16 @@ export async function verifyBlocksExecutionPayload(
     }
   }
 
+  if (blocks.length === 1 && opts.seenTimestampSec !== undefined) {
+    const delaySec = chain.clock.secFromSlot(blocks[0].message.slot, opts.seenTimestampSec);
+    const recvToVerifiedExecPayload = chain.clock.secFromSlot(blocks[0].message.slot, Date.now() / 1000) - delaySec;
```

Comment:
Isn't this value equivalent to `const recvToVerifiedExecPayload = Date.now() / 1000 - opts.seenTimestampSec`?

Reply:
yeah that'd make it simpler 👍